src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.targets

```diff
@@ -128,6 +128,12 @@ Copyright (C) 2012 Xamarin Inc. All rights reserved.
 		/>
 	</CreateProperty>
 
+  	<CreateProperty Value="$(_JavaSdkDirectory)bin">
+		<Output TaskParameter="Value" PropertyName="JavacToolPath"
+				Condition="'$(JavacToolPath)' == ''"
```

Comment:
Should this also have `And Exists('$(_JavaSdkDirectory)bin')`?  Or will we error out earlier if `$(_JavaSdkDirectory)` doesn't exist?

Reply:
…and shouldn't we start using `<PropertyGroup/>` instead of `<CreateProperty/>` here?

Reply:
I did this to that it matches what was already there. I can rework this to use `PropertyGroup`. Or perhaps we get this in, and then do a PR to remove ALL `CreateProperty` references?